PWGHF/Tasks/taskPidStudies.cxx

```diff
@@ -18,6 +18,7 @@
 /// \author Luca Aglietta <luca.aglietta@unito.it>, Università and INFN Torino
 
 #include "TPDGCode.h"
+#include "TMath.h"
```

Comment:
```suggestion
```

Reply:
Yes it's not needed anymore, I removed it, thank you!